experimental/saiexperimentaldashoutboundcatopa.h

```diff
@@ -243,6 +243,18 @@ typedef enum _sai_outbound_ca_to_pa_entry_attr_t
      */
     SAI_OUTBOUND_CA_TO_PA_ENTRY_ATTR_IP_ADDR_FAMILY,
 
+    /**
+     * @brief Action parameter outbound port map id
+     *
+     * @type sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_OUTBOUND_PORT_MAP
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
+     * @validonly SAI_OUTBOUND_CA_TO_PA_ENTRY_ATTR_ACTION == SAI_OUTBOUND_CA_TO_PA_ENTRY_ACTION_SET_PRIVATE_LINK_MAPPING
+     */
+    SAI_OUTBOUND_CA_TO_PA_ENTRY_ATTR_OUTBOUND_PORT_MAP_ID,
+
```

Comment:
Do we need SAI_OUTBOUND_CA_TO_PA_ENTRY_ATTR_SVC_SRC_PREFIX and SAI_OUTBOUND_CA_TO_PA_ENTRY_ATTR_SVC_DST_PREFIX as well?

Reply:
Updated attributes to include them.